### ISSUE TITLE
delegate_to prefers connection and remote_user from play instead of host vars

### ISSUE THREAD

**Issue by ibratoev** [ansible/ansible#10661]
##### Issue Type:

Bug Report - regression from 1.8.4
##### Ansible Version:

1.9.0.1
1.9.1RC1
##### Environment:

Ubuntu 14.04 controller
Windows 2008R2SP1 host
##### Summary:

Example is worth a thousand words:

```
- name: Example play
  hosts: localhost
  connection: local
  tasks:
    - name: Ping test
      delegate_to: winhost
      #connection: winrm
      #remote_user: Administrator
      raw: dir
```

Additionally I have the following variables defined for a group **winhost** is part of:

```
ansible_ssh_user: USER
ansible_ssh_pass: PASS
ansible_connection: winrm
```

These variables are defined in let's say: _group_vars/win-group.yml_.
##### Steps To Reproduce:

Run the provided example.
##### Expected Results:

**delegate_to** picks the connection information from the group_vars:

```
ansible_ssh_user: USER
ansible_ssh_pass: PASS
ansible_connection: winrm
```

and connects to the host successfully.
##### Actual Results:

_401 Unauthorized. basic auth failed_

If you uncomment the commented lines:

```
      #connection: winrm
      #remote_user: Administrator
```

everything starts to work.

In Ansible 1.8.4 everything works fine as expected.

**Comment by bcoca** [comment]
see if #10701 fixes this

**Comment by jimi-c** [comment]
The above was merged in and will be included in 1.9.1, so going to go ahead and close this. Please let us know if you continue seeing problems related to this.

Thanks!
